### PR TITLE
new error pages

### PR DESCRIPTION
**Pull request by johnnyporkchops**
New 404 and 500 pages based on https://github.com/18F/fec-cms/issues/966
It was suggested in the conversation to change "previous FEC.gov" to "old FEC.gov design." If this needs to be changed back, let me know.

### CODE REVIEW

File: fec/fec/settings/base.py [hunk 1]
```diff
@@ -172,6 +172,8 @@
 from fec import constants
 from .env import env
 
+WEBMANAGER_EMAIL = "webmanager@fec.gov"
+CLASSIC_URL = "http://www.fec.gov"
```
Comment (noahmanger) [inline fec/fec/settings/base.py]: This is already stored below as`FEC_CLASSIC_URL`.

File: fec/fec/templates/404.html [hunk 2]
```diff
@@ -5,16 +5,13 @@
...
-          <li><a class="button button--alt" href="https://github.com/18f/fec/issues">File an issue</a></li>
-          <li><a class="button button--alt" href="mailto:{{ settings.CONTACT_EMAIL }}">Email our team</a></li>
-        </ul>
-      </div>
+      <h1 class="message__title">Page not found</h1>
+      <p>Sorry, we couldn't find the page you're looking for.</p>
+      <p>FEC.gov is in the middle of a redesign. We've worked hard to keep our URLs up to date, but please let us know if you think something is missing or broken. We're available by <a href="mailto:{{ settings.WEBMANAGER_EMAIL }}">email</a> or <a href="https://github.com/18f/fec">GitHub</a>.</p>
+      
+        <p><a class="button button--cta" href="/">Return home</a></p>
+        <p>or try searching an archive of the <a href="{{settings.CLASSIC_URL}}">old FEC.gov design.</a></p>
```
Comment (noahmanger) [inline fec/fec/templates/404.html]: @emileighoutlaw should this O in "or" be capitalized? I think it looks a little weird lowercase (though I know this is how it was in the design):

image

File: fec/fec/settings/base.py (hunk 1 above)
Comment (patphongs) [inline fec/fec/settings/base.py]: @noahmanger @emileighoutlaw The classic domain has already been set up. So you can point the url to be `CLASSIC_URL = "http://classic.fec.gov"`

Comment (johnnyporkchops) [inline fec/fec/settings/base.py]: Should email for error pages go to betafeedback@fec.gov  or webmanager@fec.gov ?

Comment (emileighoutlaw) [inline fec/fec/settings/base.py]: email should be webmanager@fec.gov

**Comment by emileighoutlaw** [comment]
The content in this looks good to me!

**Comment by johnnyporkchops** [comment]
### Changes Made, PLZ review:
cms_404_error

**Pull request merged**